test/common/config.h

```diff
@@ -74,4 +74,16 @@ const unsigned MByte = 1024*1024;
 #define __TBB_TEST_SKIP_AFFINITY 1
 #endif
 
+#ifdef __has_feature
+	#define __TBB_HAS_FEATURE(x) __has_feature(x)
+#else
+	#define __TBB_HAS_FEATURE(x) 0
+#endif
+
+#ifdef __SANITIZE_ADDRESS__
+	#define __TBB_TEST_USE_ADDRESS_SANITIZER 1
+#else
+	#define __TBB_TEST_USE_ADDRESS_SANITIZER __TBB_HAS_FEATURE(address_sanitizer)
+#endif
```

Comment:
```suggestion
#ifdef __SANITIZE_ADDRESS__
	#define __TBB_TEST_USE_ADDRESS_SANITIZER 1
#elif defined(__has_feature)
#if __has_feature(address_sanitizer)
      #define __TBB_TEST_USE_ADDRESS_SANITIZER 1
#endif
#endif
```

Reply:
Possible "undef" warning for __TBB_TEST_USE_ADDRESS_SANITIZER define in case using clang without address_sanitizer.

Reply:
could you please check it ? 
As similar check here https://github.com/oneapi-src/oneTBB/blob/master/include/oneapi/tbb/detail/_config.h#L438-L444 seems to not produce warnings

Reply:
It's not a real issue, since warring "undef" is switched off by default.